test/GitcoinGovernor.t.sol

```diff
@@ -29,3 +36,216 @@ contract GitcoinGovernorTest is Test, DeployInput {
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      string[] memory _signatures,
+      bytes[] memory _calldatas
+    ) = governorAlpha.getActions(proposalId);
+    assertEq(_targets.length, 2);
+    assertEq(_targets[0], TIMELOCK);
+    assertEq(_targets[1], address(governor));
+    assertEq(_values.length, 2);
+    assertEq(_values[0], 0);
+    assertEq(_values[1], 0);
+    assertEq(_signatures.length, 2);
+    assertEq(_signatures[0], "setPendingAdmin(address)");
+    assertEq(_signatures[1], "__acceptAdmin()");
+    assertEq(_calldatas.length, 2);
+    assertEq(_calldatas[0], abi.encode(address(governor)));
+    assertEq(_calldatas[1], "");
+  }
+
+  function test_proposalActiveAfterDelay() public {
+    jumpToActiveProposal();
+
+    // Ensure proposal has become active the block after the voting delay
+    uint8 _state = governorAlpha.state(proposalId);
+    assertEq(_state, ACTIVE);
+  }
+
+  function testFuzz_ProposerCanCastVote(bool _willSupport) public {
+    jumpToActiveProposal();
+    uint256 _proposerVotes = gtcToken.getPriorVotes(PROPOSER, proposalStartBlock());
+
+    vm.prank(PROPOSER);
+    governorAlpha.castVote(proposalId, _willSupport);
+
+    IGovernorAlpha.Receipt memory _receipt = governorAlpha.getReceipt(proposalId, PROPOSER);
+    assertEq(_receipt.hasVoted, true);
+    assertEq(_receipt.support, _willSupport);
+    assertEq(_receipt.votes, _proposerVotes);
+  }
+
+  function test_ProposalSucceedsWhenAllDelegatesVoteFor() public {
+    jumpToActiveProposal();
+
+    // All delegates vote in support
+    for (uint _index = 0; _index < delegates.length; _index++) {
+      vm.prank(delegates[_index]);
+      governorAlpha.castVote(proposalId, true);
+    }
+
+    jumpToVoteComplete();
+
+    // Ensure proposal state is now succeeded
+    uint8 _state = governorAlpha.state(proposalId);
+    assertEq(_state, SUCCEEDED);
+  }
+
+  function test_ProposalDefeatedWhenAllDelegatesVoteAgainst() public {
+    jumpToActiveProposal();
+
+    // All delegates vote against
+    for (uint _index = 0; _index < delegates.length; _index++) {
+      vm.prank(delegates[_index]);
+      governorAlpha.castVote(proposalId, false);
+    }
+
+    jumpToVoteComplete();
+
+    // Ensure proposal state is now defeated
+    uint8 _state = governorAlpha.state(proposalId);
+    assertEq(_state, DEFEATED);
+  }
+
+  function test_ProposalCanBeQueuedAfterSucceeding() public {
+    passProposal();
+    governorAlpha.queue(proposalId);
+
+    // Ensure proposal can be queued after success
+    uint8 _state = governorAlpha.state(proposalId);
+    assertEq(_state, QUEUED);
+
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      string[] memory _signatures,
+      bytes[] memory _calldatas
+    ) = governorAlpha.getActions(proposalId);
+
+    uint256 _eta = block.timestamp + timelock.delay();
+
+    for (uint _index = 0; _index < _targets.length; _index++) {
```

Comment:
Per CI, needs formatting

> error: Formatting validation failed, run `scopelint fmt` to fix

(`forge fmt` also works, but I noticed these uints and I know you're a uint256 maxi 😛)

Reply:
Good catch! I think this snuck in here because I used my editor autocomplete. Fixed in e16b7ce09fd3993ea46cd50337e069a5b6efc621

Reply:
I think you only fixed the uint but didn't actually run `forge fmt`, since that check is still failing

Reply:
Gonna do a separate PR to update all formatting now that `scopelint` is upgrading. Don't want the diff noise on this one.